docs/DeveloperGuide.md

```diff
@@ -1,9 +1,98 @@
 ---
 layout: page
 title: Developer Guide
+nav-text: Developer Guide
 ---
-* Table of Contents
-{:toc}
+<p align="center">
+  <img src="https://i.imgur.com/dLt7IDY.png" />
+</p>
+
+<p align="center">
+  <img src="https://i.imgur.com/FKI1H1U.png" width="300"/>
+</p>
+
+# CLITutors Developer Guide (v1.3)
+
+<div style="page-break-after: always;"></div>
+
+## Table of Contents {:.no_toc}
+Table of Contents {:toc}
+
+**`CLITutors`** is a desktop app for **managing private tutoring jobs**, optimized for use via a **Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI). If you have a big list of tutors to manage, `CLITutors` helps you to manage matching tutors and students for private tuition faster than using a regular database.
+
+## **Navigating this Developer Guide**
+Before diving into the rest of the contents in our developer guide, the following are a few important syntaxes to take note of to facilitate your reading:
+
+| Syntax                                                                  | Description                                         |
+| ----------------------------------------------------------------------- | --------------------------------------------------- |
+| `Markdown`                                                              | Denotes distinct classes, their methods or examples |
+| <div markdown="span" class="alert alert-info">:information_source: Note</div> | Important things to take note of                    |
+| <kbd>Keyboard</kbd>                                                     | Keyboard actions                                    |
+
+<div style="page-break-after: always;"></div>
+
+## **Acknowledgements**
+
+We would like to thank Jun Xiong and Damith for supervising our project.
+
+## **Setting up, getting started**
+
+Refer to the guide [_Setting up and getting started_](SettingUp.md).
+
+## **Design**
+
+<div markdown="span" class="alert alert-info"> :bulb: **Tip:** The `.puml` files used to create diagrams in this document can be found in the [diagrams](https://github.com/AY2122S1-CS2103T-T17-2/tp/tree/master/docs/diagrams/) folder. Refer to the [_PlantUML Tutorial_ at se-edu/guides](https://se-education.org/guides/tutorials/plantUml.html) to learn how to create and edit diagrams.
+</div>
+
+### Architecture
+
+<p align="center">
+  <img src="images/ArchitectureDiagram.png" />
```

Comment:
I think you might not have updated the png to follow the latest changes? This was the old architecture diagram if I'm not wrong